README.md

```diff
@@ -391,6 +414,7 @@ GLOBAL OPTIONS:
    --kube-username USERNAME, -u USERNAME  kubernetes auth USERNAME [$KUBE_USERNAME, $PLUGIN_KUBE_USERNAME]
    --kube-password PASSWORD, -p PASSWORD  kubernetes auth PASSWORD [$KUBE_PASSWORD, $PLUGIN_KUBE_PASSWORD]
    --config value                         Env file location [$CONFIG_FILE, $PLUGIN_CONFIG_FILE]
+   --config-data value                    Config data e.g. --config-data=Chart=./Chart.yaml [$KD_CONFIG_DATA, $PLUGIN_KD_CONFIG_DATA]
```

Comment:
Bit confused by the use as `--config-data=Chart=./Chart.yaml`. In the example further up it's just the filepath provided to the flag.

Reply:
Scoped under `Chart` it seems when looking into the code below. Probably good to include an example for this in the README also, and demonstrate how it's referred to in the deployment files.

Reply:
So --config-data value has two forms:

1. Scope=file.yaml - data from file available at .Scope.rootkey.subkey
2. file.yaml - data from file available at .rootkey.subkey

I'll try and make the docs clearer...